### ISSUE TITLE
support for T4

### ISSUE THREAD

**Issue by krishnanpooja** [NVIDIA/TensorRT-LLM#2620]
### System Info

T4 GPU
TensorRT-LLM: 0.15.0.dev2024111200

### Who can help?

@ncomly-nvidia , @juney-nvidia , @byshiue 

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

Build models for tensorrt-llm engine on T4 GPU.

### Expected behavior

Should work on T4 GPU 

### actual behavior

I am seeing warning message
[12/24/2024-10:21:22] [TRT-LLM] [W] Failed to infer cluster info for Tesla T4, treat it as a L40 node with 15 GB memory. This setting makes no effect if you do not use auto parallel.

How do I get it to recognize T4?

### additional notes

The model works on A100. Both single and mGPU setup

**Comment by nv-guomingz** [comment]
Hi @krishnanpooja , T4 is no longer a support hardware since 0.14 release.

**Comment by krishnanpooja** [comment]
@nv-guomingz, thank you!
May I ask, why is T4 no longer supported?

**Comment by nv-guomingz** [comment]
> [@nv-guomingz](https://github.com/nv-guomingz), thank you! May I ask, why is T4 no longer supported?

Hi @krishnanpooja  thanks for your patience.

The reason we don't support T4 any more is because the lack of resource.

Please feel free to reopen it if needed.

**Comment by jingzhaoou** [comment]
With the proliferation of many SLMs like Llama-3.2 1B/3B models, T4 becomes a very attractive option cost and scalability wise. I sincerely hope TensorRT-LLM can resume T4 support. Can anyone help re-open this issue? Thanks a lot!

**Comment by bulatovv** [comment]
Yeah, T4 is really handy for running smaller models. It would be awesome if you could support these GPUs again — they’re still very useful for a lot of people. Thanks!

**Comment by bulatovv** [comment]
Yeah, T4 is really handy for running smaller models. It would be awesome if you could support these GPUs again, they’re still very useful for a lot of people. Thanks!
